app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+    @user = current_user
+  end
+
+  def create
+    @post = Post.find(params[:post_id] || params[:id])
+    @like = Like.new(post_id: @post.id, author_id: current_user.id)
+    @like.author_id = current_user.id
+
+    respond_to do |format|
+      if @like.save
+        format.html { redirect_to user_post_url(current_user, @post), notice: 'Like was successfully created.' }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+      end
```

Comment:
- [ ] A user should be able to add a like to posts. Currently the user cannot do so. Kindly fix it.
![image](https://user-images.githubusercontent.com/81932478/205086807-ddd5b434-8856-498e-8cc2-9e58c7450e2a.png)
- [ ] Also the posts counter is showing 2 posts but the the user has 3 posts